src/utils/hdfs_mods.py

```diff
@@ -2,12 +2,19 @@
     These functions will need to be tested separately, using mocking.
 """
 
-import pydoop.hdfs as hdfs
 import pandas as pd
 import json
 import os
 import logging
 
+try:
+    import pydoop.hdfs as hdfs
+    from src.utils.hdfs_mods import read_hdfs_csv, write_hdfs_csv
+    HDFS_AVAILABLE = True
+except ImportError:
+    HDFS_AVAILABLE = False
+
+# set up logging
```

Comment:
How is HDFS_AVAILABLE used?

Reply:
It is not currently, but It can be used later if other functions need to load things differently (I think they will).